docs/workforce-management-framework/topics/con-datasync-module.adoc

```diff
@@ -0,0 +1,39 @@
+= {WFM-RC-NameLong} Datasync
+
+As with all synchronisation tasks Datasync is made up of a cloud and client sides. 
+The cloud is implemented with the Datasync cloud module and client is implemented with the Datasync client module. 
+These are a implementation of the Sync Framework for advanced usage see the 
+link:{WFM-RC-SyncFrameworkURL}{WFM-RC-Branch}/docs[sync framework documentation]. 
+The primary funtion of Datasync is to synchronise mobile data between mobile(client) and a backend server(cloud).
```

Comment:
@wtrocki - are you ok with the update of terms - basically I'm changing this:
 - "The primary funtion of Datasync is to synchronise mobile data between mobile(client) and a backend server(cloud)."

to this:

- "The primary function of *Datasync* is to synchronize data between the Client Apps (mobile) and the Cloud App (running on a backend server)."

It's more inline with our existing documentation - ie: the identification of a Client App (mobile) and a Cloud App (running on a back end server).

NOTE: It'll have a knock on affect to the rest of the doc that I'll handle.